bool use_stored_footstep_plan
controller_msgs/FootstepDataListMessage planned_footsteps

---

bool success
